demo/tabs-basic-demos.html

```diff
@@ -98,6 +98,24 @@ <h3>Pre-selected tab</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Anchors</h3>
+    <p>
+      <b>Note:</b> wrapping tab content into <code>&lt;a&gt;</code> introduces a limitation of styling slotted
+      elements, like icons, by the <code>&lt;vaadin-tabs&gt;</code> theme.
+    </p>
+    <vaadin-demo-snippet id='tabs-basic-demos-anchors'>
+      <template preserve-content>
+        <vaadin-tabs>
+          <vaadin-tab>
+            <a href="#tabs-integration-demos">Integration</a>
```

Comment:
Need to use some other links here, these won't work on vaadin.com

Reply:
Should also use `tabindex="-1"`. Now it takes 2 tab hits to escape the tabs

Reply:
Done.